bin/README.md

```diff
@@ -0,0 +1,29 @@
+# funclist
+
+usage:
+```
+./funclist api.json
+```
+
+Look in a OpenApi json file, the list of Functions.
+To do so, look at which object in components.schemas end up with Request.
+Then each functions call are separate by a space
```

Comment:
Look in an OpenAPI JSON file for the list of functions. To do this, identify which objects in components.schemas end with 'Request'. Each function call is separated by a space.